app/objective/get.controller.js

```diff
@@ -0,0 +1,27 @@
+const { logger } = require('../../common/logging/logger')
+const { getSentencePlanObjective } = require('../../common/data/sentencePlanningApi')
+const { removeUrlLevels } = require('../../common/utils/util')
+
+const getObjective = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planId, objectiveId }, session: { 'x-auth-token': token } },
+  res
+) => {
+  const nexturl = path.substring(0, path.lastIndexOf('/'))
+  const backurl = removeUrlLevels(path, 2)
+  const renderDetails = { ...renderInfo, nexturl, backurl }
+
+  if (body.objective) {
+    renderDetails.objective = body.objective
+  }
```

Comment:
This is unnecessary.
The `objective` value will be set in the object passed to the render method anyway because we deconstruct it into that object (line 20).